client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/index.tsx

```diff
@@ -1,185 +1,129 @@
-												components: {
-													a: <a href={ destSiteUrl } target="_blank" rel="noreferrer" />,
-												},
-											}
-										) }
-									</>
-								) }
-							</>
-						) }
-						{ siteMigrationKey && ! siteMigrationKeyError && (
-							<>
-								{ translate(
-									'Click {{strong}}Copy key{{/strong}} below to copy your migration key - you will need that in a few minutes to start the migration. This key is unique to your site and will only be available once.',
-									{
-										components: {
-											strong: <strong />,
-										},
-									}
-								) }
-								<div className="site-migration-instructions__migration-key">
-									<code className="site-migration-instructions__key">
-										{ hideSiteMigrationKey
-											? '*'.repeat( siteMigrationKey.length - 5 )
-											: siteMigrationKey }
-									</code>
-									<Button
-										onClick={ () => {
-											setHideSiteMigrationKey( ! hideSiteMigrationKey );
-										} }
-										className="site-migration-instructions__show-key-button components-button"
-									>
-										{ hideSiteMigrationKey ? translate( 'Show key' ) : translate( 'Hide key' ) }
-									</Button>
-									<ClipboardButton
-										text={ siteMigrationKey }
-										className="site-migration-instructions__copy-key-button is-secondary"
-										onCopy={ onCopy }
-									>
-										{ buttonTextCopy ? translate( 'Copied!' ) : translate( 'Copy key' ) }
-									</ClipboardButton>
-								</div>
-							</>
-						) }
-					</div>
-				</li>
 				<li>
 					{ translate(
-						'Go to the {{a}}Migrate Guru page on the source site{{/a}}, enter your email address, and click {{strong}}Migrate{{/strong}}.',
+						'Install and active the {{a}}Migrate Guru plugin{{/a}} on your existing site.',
 						{
 							components: {
-								a: <a href={ sourceSiteUrl } target="_blank" rel="noreferrer" />,
-								strong: <strong />,
+								a: <a href={ getInstallationURL( fromUrl ) } target="_blank" rel="noreferrer" />,
 							},
 						}
 					) }
 				</li>
 				<li>
 					{ translate(
-						'You will see a screen showing multiple hosting providers. Select {{em}}Automattic{{/em}} as the destination host.',
+						'Go to the {{a}}Migrate Guru page on the source site{{/a}}, enter your email address, and click {{strong}}Migrate{{/strong}}.',
 						{
 							components: {
-								em: <em />,
+								a: <a href={ getMigrateGuruPageURL( fromUrl ) } target="_blank" rel="noreferrer" />,
+								strong: <strong />,
 							},
 						}
 					) }
 				</li>
 				<li>
-					{ translate(
-						'Find the {{em}}Migrate Guru Migration Key{{/em}} field, and paste the key you copied in step 2. Then click {{strong}}Migrate{{/strong}} to start your migration.',
-						{
-							components: {
-								em: <em />,
-								strong: <strong />,
-							},
-						}
-					) }
+					{ translate( 'When asked to select a destination host, pick {{em}}“Automattic”{{/em}}.', {
```

Comment:
Can we set the name of the host to "WordPress.com", they already updated it.

Reply:
Updated here  cc012deec311e73a0a9e059bd637364e965f9f60